pkg/gitops/generate_build.go

```diff
@@ -123,6 +133,24 @@ func DetermineBuildExecution(component appstudiov1alpha1.Component, params []tek
 	return pipelineRunSpec
 }
 
+// Tries to load a custom build bundle path from a configmap.
+// The following priority is used: component's namespace -> default namespace -> fallback value.
+func resolveBuildBundle(component appstudiov1alpha1.Component) string {
+	namespaces := []string{component.Namespace, BuildBundleDefaultNamepace}
```

Comment:
nitpick: I don't think that a slice is needed here.

Reply:
Any suggestions? I did this to avoid duplicating the code in L144-L148.

Reply:
nitpicking really. An array would be a bit more performant as you don't intend to extend the size:
namespaces := [2]string{component.Namespace, BuildBundleDefaultNamepace}